resources/translations/en.edn

```diff
@@ -218,36 +218,42 @@
   :dropdown {:no-results "No results"
              :placeholder "Select..."}
-  :email {:application-approved {:subject "Application approved (%3)"
-                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been approved.\n\nView application: %6"
-                                 :message-to-handler "Dear %1,\n\n%2 has approved the application %3 from %4.\n\nView application: %6"}
-          :application-rejected {:subject "Application rejected (%3)"
-                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been rejected.\n\nView application: %6"
-                                 :message-to-handler "Dear %1,\n\n%2 has rejected the application %3 from %4.\n\nView application: %6"}
-          :application-closed {:subject "Application closed (%3)"
-                               :message-to-applicant "Dear %1,\n\nYour application %3 has been closed.\n\nView application: %6"
-                               :message-to-handler "Dear %1,\n\n%2 has closed the application %3 from %4.\n\nView application: %6"}
-          :application-returned {:subject "Application returned (%3)"
-                                 :message-to-applicant "Dear %1,\n\nYour application %3 has been returned.\n\nView application: %6"
-                                 :message-to-handler "Dear %1,\n\n%2 has returned the application %3 from %4.\n\nView application: %6"}
-          :application-licenses-added {:subject "Terms of use added to application (%3)"
-                                       :message-to-applicant "Dear %1,\n\nTerms of use have been added to your application %3. They need to be accepted before gaining access to the resources.\n\nView application: %6"
-                                       :message-to-handler "Dear %1,\n\n%2 has added terms of use to the application %3 from %4.\n\nView application: %6"}
-          :application-submitted {:subject "Application submitted (%4: %3)"
-                                  :message "Dear %1,\n\n%4 has submitted an application (%3): %5.\n\nView application: %6"}
-          :comment-requested {:subject "Review requested (%2: %3)"
-                              :message "Dear %1,\n\n%2 has requested a review on application %3.\n\nComment: %6"}
-          :commented {:subject "Review added (%2: %3)"
-                      :message "Dear %1,\n\n%2 has reviewed application %3.\n\nView application: %6"}
-          :decided {:subject "Decision made (%2: %3)"
-                    :message "Dear %1,\n\n%2 has made a decision on application %3.\n\nView application: %6"}
-          :decision-requested {:subject "Decision requested (%2: %3)"
-                               :message "Dear %1,\n\n%2 has requested your decision on application %3.\n\nView application: %6"}
-          :footer "\n\nPlease don't reply to this automatically generated message."
+  :email {:application-approved {:subject-to-applicant "Your application has been approved (%3)"
```

Comment:
Was the id/description information supposed to be removed from the subject lines, or did you decide yourself to keep it?

Reply:
the THL materials don't include it, but I decided that it can't hurt

Reply:
I guess it's fine.